crema/qvalues.py

```diff
@@ -216,6 +231,8 @@ def estimate_pi0(pval_list):
     maxLambda = 0.5
     numBoot = 100
 
+    # LOGGER.debug("pval_list=%s", pval_list)
```

Comment:
I see there are a couple of places where you commented out some logger stuff. Can I assume that was an oversight and not meant to mean anything?

Reply:
I found the logging statements useful for debugging, but even debug-level logging seems to be disallowed when `numba.njit` is in use.

Reply:
Got it thanks. That was helpful.